### DISCUSSION TITLE
Ability to Star Shows or Movies without Requesting them

### DISCUSSION THREAD

**Discussion by FlamingBluejay** [seerr-team/seerr#427] [Ideas]
A feature that I would find particularly useful would be a way to star either shows or movies so that someone could save them for later in a list without a need to request them. This could be useful if the Jellyfin media library is getting full, or the show or movie is on a platform that I already have access to such as Netflix. This is useful because if that platform removes that show or movie, I no longer would have access to it and have to manually remember it and search for it to find it again. 

Additionally, an idea that I think could be easy to implement, through a setting, is to ask what streaming platforms the user has access to. For example, if they have access to Netflix, Amazon Prime Video, Disney Plus, or other major streaming networks, the movies or shows might prompt the user if they would like to download the media anyway despite already having access to it. If the user chooses not to download it, maybe it could be added to the list I mentioned earlier. Think of it as a way to make a "To Watch List" of all the movies and shows that person intends to see without the need to manually create a list themselves. It would be faster, look better, and would ensure that they have the correct piece of film, while also streamlining the process of keeping track of media between Jellyfin and the paid streaming platforms.

**Comment by fallenbagel** [comment]
You can already star items (the feature is in development branch rn) as watchlist
